src/pepr/operator/controllers/istio/istio-resources.ts

```diff
@@ -105,7 +105,7 @@ export async function istioResources(pkg: UDSPackage, namespace: string) {
  */
 export async function istioEgressResources(pkg: UDSPackage, namespace: string) {
   // Get package data
-  const istioMode = pkg.spec?.network?.serviceMesh?.mode || Mode.Sidecar;
+  const istioState = getIstioStateFromPackage(pkg);
```

Comment:
My fault on the comment before - I think we can eliminate `getIstioStateFromPackage` as a helper func and just use `Mode` similar to what you had here before:
```ts
   const istioMode = pkg.spec?.network?.serviceMesh?.mode || Mode.Sidecar;
```
Intent with my previous comment was more to make sure that our functions like `reconcileSharedEgressResources` have a `Mode` type param, rather than string, and then aligning all the places we check mode with that (rather than the bool for ambient in some spots).

Reply:
LGTM, thanks!